cumulus_library/module_allowlist.json

```diff
@@ -2,6 +2,8 @@
     "desc" : 
         "this dict maps pip installable studies to expected paths inside site-packages",
     "allowlist" : {
-        "covid_symptom" : "cumulus_library_covid/covid_symptom"        
+        "covid_symptom" : "cumulus_library_covid/covid_symptom",
+        "data_metrics" : "cumulus_library_data_metrics",
+        "umls" : "cumulus_library_umls/umls"
```

Comment:
Data Metrics isn't available yet on pip, but it will be soon. I'm using the no-subdir pathing for it, which will require a library release to make happen (but that's planned I believe).

I figured I'd also add umls here (with the subdir, since that's what it uses now).

No other studies are in pypi, yeah?